CMakeLists.txt

```diff
@@ -378,19 +384,23 @@ install(
 install(
     FILES
         "${CMAKE_CURRENT_BINARY_DIR}/OpenSnConfig.cmake"
+        "${CMAKE_CURRENT_BINARY_DIR}/opensnConfig.cmake"
```

Comment:
You may want to remove this line as well...

Reply:
Yep, missed that. Fixed.